digitalpublicgoods/provenance-services-with-smart-data-agreements-using-data-exchange-protocols.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Provenance Services with Smart Data Agreements using Data Exchange Protocols",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Unknown",
+    "copyrightURL": ""
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/decentralised-dataexchange/data-exchange-agreements/tree/main"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR"
+    ],
+    "adherenceSteps": [
```

Comment:
Check with project whether the Uses Data Protection Impact Assessment can be made public. Thanks.

Reply:
Please note that the DPIA is owned by organisations that are part of the ecosystem. This is an open-source code for anyone to use and be part of a data exchange ecosystem.